src/components/SliderBox/components/SliderArrow/SliderArrow.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
```

Comment:
import two items from @mui/icons-material

Reply:
It`s svg icons exported by default  in lib , i tried to do that, but i got error, only this was was correct

Reply:
discussed approach at the meeting

Reply:
and please sort imports

Reply:
fixed